docs/setup/self-hosted/docker.mdx

```diff
@@ -1,4 +1,4 @@
----
+    ---
```

Comment:
Please remove these spaces in front of `---`. Not sure but this may affect the rendering of the doc page.

Reply:
Done.